runtime/line.go

```diff
@@ -37,18 +37,23 @@ func (w *lineWriter) Write(p []byte) (n int, err error) {
 		out = w.rep.Replace(out)
 	}
 
-	line := &Line{
-		Number:    w.num,
-		Message:   out,
-		Timestamp: int64(time.Since(w.now).Seconds()),
-	}
+	lines := strings.Split(strings.TrimSpace(out), "\n")
+
+	for _, l := range lines {
+		line := &Line{
+			Number:    w.num,
+			Message:   l + "\n",
```

Comment:
we actually don't want to automatically add a `\n` because sometimes `[]byte` containers same-line output (e.g. `print` and not `println`) depending on how it is flushed by the docker daemon :)

Reply:
Sadly I'm not entirely sure on how exactly you want it to be. Do you mind simply changing it yourself, before merging? Otherwise you'd need to explain again. Maybe also send a gist of the unit test on how it should be? Thanks!